src/resource/resource.ts

```diff
@@ -176,6 +180,29 @@ export function isPodOwner(
   return podOwner === webId;
 }
 
+/**
+ *  Given an solidDataset, fetch the urls of all contained resources.
+ *  If the solidDataset given is not a container, or is missing resourceInfo, throw an error.
+ *
+ * @param solidDataset The container from which to fetch all contained resource urls
+ * @returns A list of urls, each of which points to a contained resource of the given solidDataset
+ * @since 1.2.0
+ */
+
+export function getContainedResourceUrlAll(
```

Comment:
Since this function only applies to SolidDatasets (i.e. not to Files), it would be better placed in `src/resource/SolidDataset.ts`. Functions in this file apply to either :)

(The reason this matters is because that affects where it will be listed in the documentation, and also from which module people import the file when using [package exports](https://webpack.js.org/guides/package-exports/).)